lib/stringutils/slice.go

```diff
@@ -19,6 +19,38 @@ func SubSlice(s1 []string, s2 []string) bool {
 	return true
 }
 
+// This method is used to return comman elements between the two slices
+func SliceIntersection(first, second []string) []string {
+	out := []string{}
+	bucket := map[string]bool{}
+
+	for _, i := range first {
+		for _, j := range second {
+			if i == j && !bucket[i] {
+				out = append(out, i)
+				bucket[i] = true
+			}
+		}
+	}
+
+	return out
+}
+
+// This method is used to return uncomman elements between the two slices
+func SliceDifference(a, b []string) []string {
```

Comment:
please use the proper naming conventions for variables instead of a and b. Also, the method description should be clear, in this case from which slice does this method return the uncommon elements?